index.html

```diff
@@ -0,0 +1,233 @@
+                      </svg>
+           </a>
+          </ul>
+          </div> 
+  </section>   
+    <img src="eclipse/" alt="">
+  </section>
+  <section class="container">
+    <section class="card_item">
+      <div class="snap">
+      </div>
+      <h1 class="tex"> Tonic</h1> <br>
+      <ul class="list">
+        <li class="canopy"><b> CANOPY</b></li>
+        <li ><img src="images/cano/Counter.png" alt=""></li>
+        <li class="bac">Back End Dev</li>
+        <li><img src="images/cano/Counter.png" alt=""></li>
+        <li class="bac">2015</li>
+      </ul>
+      <p class="p"> A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+       <ul class="text">
+         <li class="lists">html</li>
+         <li class="lists">css</li>
+         <li class="lists">javascript</li>
+       </ul> 
+       <button id="bttn">see project</button>
+    </section>
+    <section class="card_item">
+      <div class="snap">
+        <img src="images/snap2/Snapshoot Portfolio.png" alt="">
+      </div>
+      <h1 class="tex">Multi-Post Stories</h1>
```

Comment:
- [ ] Kindly I recommend you to use `hx` header tag instead of `h1` tag for "project title" for purpose of SEO optimization, All of the important pages on your site should have one `H1` tags to draw in the reader and give a clear indication of the content on the page.